### DISCUSSION TITLE
How to set NVIM_APPNAME=something in Windows powershell or cmd

### DISCUSSION THREAD

**Discussion by lakshits11** [neovim/neovim#26674] [Q&A]
### Problem

like in linux we do above thing, how to manage multiple configs in windows also

### Steps to reproduce

just asking

### Expected behavior

Seeking the solution

### Neovim version (nvim -v)

0.9.4

### Vim (not Nvim) behaves the same?

yes

### Operating system/version

Windows 11

### Terminal name/version

Windows Terminal

### $TERM environment variable

cannot see, but i know its true color

### Installation

choco install neovim

**Comment by glacambre** [comment]
Does `set NVIM_APPNAME=something` do what you want?

**Reply by wookayin** [reply]
This seems to be a correct one in windows

https://superuser.com/questions/79612/setting-and-getting-windows-environment-variables-from-the-command-prompt

**Reply by wookayin** [reply]
@lakshits11 You can verify the environment variable was set correctly by printing `:echo $NVIM_APPNAME` within nvim.

**Comment by lakshits11** [comment]
No, obviously not. If it worked, why would i have asked

**Comment by GiantSlow** [comment]
It's not the most elegant solution but it works for me.
If you're using Powershell, insert ``--cmd "let `$NVIM_APPNAME='something'"`` when you call nvim in the terminal.

**Reply by GiantSlow** [reply]
Actually, calling `$env:NVIM_APPNAME = 'something'` before calling nvim seems to work better in powershell.
source: https://petri.com/powershell-set-environment-variable/
